### DISCUSSION TITLE
Hyperapp + .NET

### DISCUSSION THREAD

**Discussion by calin-dobos** [jorgebucaran/hyperapp#1118] [Show and tell]
Hello everyone,

In the last couple of years I have created a .NET full stack framework that is inspired by Hyperapp and also uses Hyperapp as one of the core components. 

https://github.com/qwebsolutions/metapsi

The target was for everything to be self-contained, which led me on a path of dynamically generated JavaScript. For this I needed a really minimal, embeddable js framework, which made Hyperapp basically the only option. It turned out to be a perfect fit.

I also applied the 'view function of a full data model' idea for server-side rendering and it seems to result in a simple, unified development experience.

So... thanks @jorgebucaran

At this point the framework is used for some small commercial projects, including a landing page and a Ionic PWA. There is no particular roadmap for now, but development will continue for the foreseeable future.

**Comment by jorgebucaran** [comment]
Looks fun! Thanks for sharing. 😌
